src/elements/content-preview/ContentPreview.js

```diff
@@ -700,8 +700,7 @@ class ContentPreview extends React.PureComponent<Props, State> {
         }
 
         const fileOpts = { ...fileOptions };
-        const typedId: string = getTypedFileId(fileId);
-        const token: TokenLiteral = await TokenService.getReadToken(typedId, tokenOrTokenFunction);
+        const token = typedId => TokenService.getReadTokens(typedId, tokenOrTokenFunction);
```

Comment:
Do we even need the argument in the token function? Since we know what file we're generating the token for, can we instead do `const token = () => TokenService.getReadToken(typedId, tokenOrTokenFunction)`?

Reply:
If I get rid of the argument, I have to change the preview lib's `getToken` function, which I don't think we should touch.